contributor-docs/maintainers-guide.md

```diff
@@ -126,4 +128,45 @@ To run the configuration script:
         -s 1
       ```
 
-**Note**: To see optional configurable parameters, run `./configure-runners.sh -h`
\ No newline at end of file
+    **Note**: To see optional configurable parameters, run `./configure-runners.sh -h`
+
+#### CloudSQL proxy server VM
+
+Datastream integration tests (IT's) that connect to a CloudSQL database rely on the 
+[Cloud auth proxy](https://cloud.google.com/sql/docs/mysql/sql-proxy) to make a connection profile. Due to connection
+issues related to which port the proxy server is hosted on, there is a separate workflow for provisioning the
+dedicated proxy server VM.
```

Comment:
Are we able to run the tests without the dedicated proxy VM? If they're flaky, that's ok - I want to make sure folks have a way to run tests locally though (not just through CI)

Reply:
If this needs to be done regardless, then we should allow projects other than `cloud-teleport-testing` below